### PR TITLE
Component retag

### PR DESCRIPTION
**Pull request by akang0**

### CODE REVIEW

**Comment by gurnben** [comment]
@schmidtd any objections?

File: modules/component/Makefile [hunk 1]
```diff
@@ -96,6 +96,18 @@ component/push: %push:
 	$(DOCKER) push ${COMPONENT_DOCKER_REPO}/${COMPONENT_NAME}:${COMPONENT_VERSION}${COMPONENT_TAG_EXTENSION}
 	@echo "Pushed the following image: ${COMPONENT_DOCKER_REPO}/${COMPONENT_NAME}:${COMPONENT_VERSION}${COMPONENT_TAG_EXTENSION}"
 
+.PHONY: component/retag
+## Push the component to COMPONENT_DOCKER_REPO
```
Comment (ycao56) [inline modules/component/Makefile:100]: wrong comment

File: modules/component/Makefile [hunk 2]
```diff
@@ -96,6 +96,18 @@ component/push: %push:
...
+component/retag: %retag:
```
Comment (ycao56) [inline modules/component/Makefile:101]: can we just call it `tag`?

**Comment by ycao56** [comment]
@gurnben @schmidtd Please help merge this so that we can continue our test automation work if the change look good to you. Thanks.

**Pull request merged**